app/app.go

```diff
@@ -1220,8 +1228,14 @@ func (app *Evmos) setupUpgradeHandlers() {
 		// no store upgrade in v9 or v9.1
 	case v10.UpgradeName:
 		// no store upgrades in v10
+	case v11.UpgradeNameRC3:
+		// rename recovery store
+		storeUpgrades = &storetypes.StoreUpgrades{
+			Deleted: []string{"recovery"},
+			Added:   []string{recoverytypes.StoreKey},
+		}
 	case v11.UpgradeName:
-		// add ica host submodule in v11
+		// add ica host submodule
 		// initialize recovery store
 		storeUpgrades = &storetypes.StoreUpgrades{
 			Added: []string{icahosttypes.SubModuleName, recoverytypes.StoreKey},
```

Comment:
Just realized won't this break mainnet if we don't change it to `ModuleName` ? @fedekunze 
```suggestion
			Added: []string{icahosttypes.SubModuleName, recoverytypes.ModuleName},
```

Reply:
@Vvaradinov upgrading from v10.0.1 to this branch works fine